[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\andye\\AppData\\Local\\Temp\\quarto-session4a5d0fc0cf976f5e'\n"]
